.gitignore

```diff
@@ -16,6 +16,7 @@
 #===============================================================================
 
 build
+build_acl
```

Comment:
If it helps, you can use build* so that any directory starting with `build` prefix would be ignored.

Reply:
Sure @mgouicem, thanks. But I have restored gitignore file in the next commit.